util/helpers/get_column_stats.sql

```diff
@@ -0,0 +1,8 @@
+CREATE OR REPLACE FUNCTION pganalyze.get_column_stats() RETURNS TABLE(
```

Comment:
Since the return type changed in #610, `CREATE OR REPLACE FUNCTION` can raise an error. We should probably use `DROP FUNCTION IF EXISTS` across the board to avoid errors like that in the future.

Reply:
Yeah, I was on the fence whether its okay to switch to `CREATE OR REPLACE FUNCTION` but I can just keep this as DROP IF EXISTS + CREATE.

Reply:
Since we're on the topic, maybe it's good to update the other helper functions?

Reply:
Sure, I can update it for consistency.